### PR TITLE
Add home page

### LINKED ISSUE #26: Create the home page
Use this for reference https://github.com/dewa-himanshu24/pandit.ai/blob/main/documentation/ui-wireframe/marked-data/Inkedpandit.ai-Home%20page.drawio.jpg

### PR DESCRIPTION
**Pull request by dewa-himanshu24**
closes #26

### CODE REVIEW

File: pages/Pooja.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+const Pooja = (props: { image: string | undefined; name: string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>> | React.ReactFragment | React.ReactPortal | null | undefined }) => {
+  return (
```
Comment (jainendra) [inline pages/Pooja.tsx:4]: remove extra types

File: pages/HomePage.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      <Navbar isLoggedIn={true} />
+      <div
+        className="relative z-10 overflow-hidden pt-[120px] pb-[100px] md:pt-[130px] lg:pt-[160px]"
+      >
+        <div className="container">
+          <div className="-mx-4 flex flex-wrap items-center">
+            <div className="w-full px-4">
+              <div className="text-center">
+                <h1 className="text-4xl font-semibold text-black">Welcome Jainendra please select a pooja to start</h1>
+              </div>
```
Comment (jainendra) [inline pages/HomePage.tsx:18]: Use props for name here

File: pages/PoojaList.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    id: "p9",
+    name: "Janmashtami",
+    image: "assets/images/blog/blog-02.jpg"
+  },
+];
+
+const PoojaList = () => {
+  const poojaList = DUMMY_POOJAS.map((pooja) => (
+    <Pooja
+      // id={pooja.id}
```
Comment (jainendra) [inline pages/PoojaList.tsx]: remove

File: pages/components/UI/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const Button = (props: { classes: string | undefined; value: string | number | readonly string[] | undefined }) => {
+  return (
```
Comment (jainendra) [inline pages/components/UI/Button.tsx:4]: remove redundant types

File: pages/Pooja.tsx (hunk 1 above)
Comment (dewa-himanshu24) [reply pages/Pooja.tsx:4]: Done

File: pages/HomePage.tsx (hunk 2 above)
Comment (dewa-himanshu24) [reply pages/HomePage.tsx:18]: Done

File: pages/PoojaList.tsx (hunk 3 above)
Comment (dewa-himanshu24) [reply pages/PoojaList.tsx]: Done

**Review by jainendra — APPROVED** [review]

**Pull request merged**